accounts/abi/bind/base.go

```diff
@@ -261,6 +266,19 @@ func (c *BoundContract) transact(opts *TransactOpts, contract *common.Address, i
 	return signedTx, nil
 }
 
+// LogEventName will verify is log belongs to the contract and return the eventName for it
```

Comment:
nit: `verify is log` -> `verify the log`

Reply:
thanks! Solved